UnityProject/Assets/Scripts/Items/Weapons/ExplosionUtils.cs

```diff
@@ -30,17 +30,25 @@ public static void PlaySoundAndShake(Vector3Int worldPosition, byte shakeIntensi
 			AudioSourceParameters audioSourceParameters = new AudioSourceParameters(0f, 100f);
 			ShakeParameters shakeParameters = new ShakeParameters(true, shakeIntensity, shakeDistance);
 
+			//Closet sound
 			_ = SoundManager.PlayNetworkedAtPosAsync(explosionSound, worldPosition, audioSourceParameters, true, false, shakeParameters);
 
-			if (groanSound?.AudioSource.OrNull()?.maxDistance == null) return;
-
-			if (shakeDistance > distantSound.AudioSource.maxDistance)
+			//Next sound
+			if (distantSound != null)
 			{
-				_ = SoundManager.PlayNetworkedAtPosAsync(distantSound, worldPosition, global: true);
+				AudioSourceParameters distantSoundAudioSourceParameters =
+					new AudioSourceParameters(0f, 100f, minDistance: 29, maxDistance: 63);
+
+				_ = SoundManager.PlayNetworkedAtPosAsync(distantSound, worldPosition, distantSoundAudioSourceParameters);
```

Comment:
what about the limitations in the network message send to nearby?

Reply:
global defaults to true, so this isnt using send to nearby